<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="10,0">
    <item row="0" column="0" colspan="6">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="pushButton_6">
      <property name="font">
       <font>
        <family>黑体</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>逆时针（90）</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton_4">
      <property name="font">
       <font>
        <family>黑体</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>缩小</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <family>黑体</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>下一张</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>黑体</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>上一张</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="font">
       <font>
        <family>黑体</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>放大</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButton_5">
      <property name="font">
       <font>
        <family>黑体</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>顺时针（90*）</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件（&amp;F）</string>
    </property>
    <addaction name="action_O"/>
    <addaction name="separator"/>
    <addaction name="action_D"/>
    <addaction name="separator"/>
    <addaction name="action_C"/>
    <addaction name="separator"/>
    <addaction name="action_P"/>
    <addaction name="separator"/>
    <addaction name="action_X_2"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑（&amp;E）</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_7"/>
    <addaction name="separator"/>
    <addaction name="action_13"/>
    <addaction name="separator"/>
    <addaction name="action_17"/>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>选择（&amp;S）</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_20"/>
    <addaction name="separator"/>
    <addaction name="action_22"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_A">
    <property name="title">
     <string>关于（&amp;A）</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="action_QT"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_S"/>
   <addaction name="menu_A"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_O">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/myImage/new1.png</normaloff>:/myImage/myImage/new1.png</iconset>
   </property>
   <property name="text">
    <string>打开（&amp;O）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_D">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/myImage/del.png</normaloff>:/myImage/myImage/del.png</iconset>
   </property>
   <property name="text">
    <string>删除（&amp;D）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_C">
   <property name="icon">
    <iconset resource="myImage.qrc">
     <normaloff>:/myImage/myImage/new2.png</normaloff>:/myImage/myImage/new2.png</iconset>
   </property>
   <property name="text">
    <string>复制（&amp;C）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_P">
   <property name="text">
    <string>粘贴（&amp;P）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>右旋转</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>下一张</string>
   </property>
  </action>
  <action name="action_22">
   <property name="text">
    <string>上一张</string>
   </property>
  </action>
  <action name="action_X_2">
   <property name="text">
    <string>退出（&amp;X）</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>左旋转</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>版本信息</string>
   </property>
  </action>
  <action name="action_QT">
   <property name="text">
    <string>关于QT</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myImage.qrc"/>
 </resources>
 <connections/>
</ui>
